target/linux/armsr/armv8/config-6.1

```diff
@@ -522,6 +522,11 @@ CONFIG_PHY_NS2_USB_DRD=y
 CONFIG_PHY_RCAR_GEN3_PCIE=y
 CONFIG_PHY_RCAR_GEN3_USB2=y
 CONFIG_PHY_RCAR_GEN3_USB3=y
+CONFIG_SERIAL_SH_SCI=y
+CONFIG_SERIAL_SH_SCI_NR_UARTS=18
+CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI_EARLYCON=y
+CONFIG_SERIAL_SH_SCI_DMA=y
```

Comment:
Please add this to `target/linux/armsr/armv8/config-6.6` too. 
Did you refresh the configuration using `make kernel_oldconfig CONFIG_TARGET=subtarget`?

Reply:
I've updated this to "target/linux/armsr/armv8/config-6.6"